docs/contributing.rst

```diff
@@ -141,6 +141,12 @@ Please take a look on the produced output.
 
 Any extra texts (print statements and so on) should be removed.
 
+Tests can also be run in parallel with:
+
+.. code-block:: shell
```

Comment:
This should actually be something like `console`. `shell` is for shell-scripts but `console` recognizes prompts (like `$` or `>`) with commands and their outputs. See the `pygments` lexers lists for more details.

Reply:
I would also mention that parallel run is experimental in the documentation unless it become stable.

Reply:
Fair enough. Though per my observations, locally it's mostly stable (unless you have a million chrome tabs plus something heavy running in the background like I do) but in CI, weird random failures will be happening more often.

Reply:
I fixed the lexer spec though it now is inconsistent will all the other lexers in the file (not sure if we want to fix this here or in another PR?) Also added a note about it being experimental.